docs/reference/commandline/dockerd.md

```diff
@@ -1279,3 +1282,25 @@ $ sudo dockerd \
         --data-root=/var/lib/docker-bootstrap \
         --exec-root=/var/run/docker-bootstrap
 ```
+
+### Default network options
+
+The `--default-network-opt` flag, and the equivalent `default-network-opts` key
+in the configuratio file, lets you specify custom default options for custom
+networks.
+
+This option accepts a value with the format `driver=opt=value`, for example:
+
+```console
+$ sudo dockerd \
+  --default-network-opt bridge=com.docker.network.bridge.host_binding_ipv4=127.0.0.1 \
+  --default-network-opt bridge=com.docker.network.bridge.mtu=1234
+```
```

Comment:
No need to change, because this matches current examples, but we should consider re-writing the examples to use `daemon.json`.

manually starting the daemon with command-line options works for debugging, but these examples can also be somewhat problematic if they're tried on a machine that already has `dockerd` running (most likely as a systemd service). In that scenario, running the example would try to start a _second_ instance of `dockerd`, which can conflict with the already running one; "best case scenario" that would produce an error, but "worst case", it may run, and potentially interfere / modify state of the running daemon.

I'm not sure what the best approach is for this though, because we also don't want to repeating "this is how you create or edit a `daemon.json` (for each platform, including Docker Desktop (?)), and this is how you start/restart/reload the daemon for testing" (in addition to some warnings "always try configuration changes in a test-environment first, etc).
 
Perhaps we need a tracking ticket for that, wdyt, @dvdksn ?

Reply:
yeah we can use a daemon.json example in this case instead

I don't think we need to go into the details of how to use this file here and everywhere else we mention it. But we should have better "entrypoint" documentation for how to work with this file. I would connect this to the grander plan of "reworking" of daemon configuration. Do we have a tracking for that?